geode-management/src/main/java/org/apache/geode/management/api/ClusterManagementOperation.java

```diff
@@ -14,12 +14,13 @@
  */
 package org.apache.geode.management.api;
 
-
 import com.fasterxml.jackson.annotation.JsonIgnore;
 
+import org.apache.geode.management.runtime.OperationResult;
+
 /**
  * Geode modules, extensions, or user code can define cluster management operations to use with
- * {@link ClusterManagementService#startOperation(ClusterManagementOperation)} by subclassing this
+ * {@link ClusterManagementService#start(ClusterManagementOperation)} by subclassing this
  * abstract base class and adding any necessary parameters. For example, a CompactDiskStore
```

Comment:
update the comment to not talk about subclassing abstract base class

Reply:
done